package.json

```diff
@@ -8,10 +9,28 @@
     "url": "git://github.com/dgaya/sinopia-memory.git"
   },
   "devDependencies": {
-    "mocha": "2.4.5"
-  },
-  "scripts": {
-    "test": "mocha ./tests"
+    "@commitlint/cli": "6.1.3",
+    "@commitlint/config-conventional": "6.1.3",
+    "@verdaccio/types": "^3.4.1",
+    "babel-cli": "6.26.0",
+    "babel-core": "6.26.3",
+    "babel-eslint": "8.2.6",
+    "babel-jest": "23.4.0",
+    "babel-plugin-flow-runtime": "0.17.0",
+    "babel-preset-env": "1.7.0",
+    "babel-preset-flow": "6.23.0",
+    "codecov": "3.0.4",
+    "cross-env": "5.2.0",
+    "eslint": "5.2.0",
+    "eslint-config-prettier": "^2.9.0",
+    "eslint-plugin-prettier": "2.6.2",
+    "flow-bin": "0.76.0",
+    "flow-runtime": "0.17.0",
```

Comment:
I think `flow-runtime` is not need it at all. The same applies to other projects. Can we do the test here?